### ISSUE TITLE
Is there any script for dividing the dataset?

### ISSUE THREAD

**Issue by YananGu** [scuwyh2000/RandBox#7]
I dont find such a script

**Comment by scuwyh2000** [comment]
> I dont find such a script

You can find the spilt in .split/ and no script for it.

**Comment by YananGu** [comment]
Thank you very much.  I dont find the test.txt for each task, like test.txt for t1, only all_task_test.txt.  Do all tasks share the same all_task_test.txt? Does it mean that I need to put the same data in t(1,2,3,4)/images/test?

**Comment by scuwyh2000** [comment]
> Thank you very much. I dont find the test.txt for each task, like test.txt for t1, only all_task_test.txt. Do all tasks share the same all_task_test.txt? Does it mean that I need to put the same data in t(1,2,3,4)/images/test?

correct

**Comment by YananGu** [comment]
I  found there are some duplicate names  in  all_task_test.txt. The original file has 10246 lines in total, but after deduplication, it only has 9119 lines. Is this normal?

**Comment by scuwyh2000** [comment]
> I found there are some duplicate names in all_task_test.txt. The original file has 10246 lines in total, but after deduplication, it only has 9119 lines. Is this normal?

yes, and this setting is following the previous work 'Towards Open World Object Detection' and 'OW-DETR: Open-world Detection Transformer'.

**Comment by YananGu** [comment]
Sorry to bother you again. In the training process, I found the log.txt is save to the result, and the model seemed to be evaluated after each task. But I cannot found the evaluation results in the log.txt. Can you tell me how to find this results in the training process? Or other methods to monitor the training process. Thank you very much.

**Comment by scuwyh2000** [comment]
> Sorry to bother you again. In the training process, I found the log.txt is save to the result, and the model seemed to be evaluated after each task. But I cannot found the evaluation results in the log.txt. Can you tell me how to find this results in the training process? Or other methods to monitor the training process. Thank you very much.

send the log to my email, scuwyh2000@gmail.com

**Comment by WangPingA** [comment]
> > 非常感谢。我没有找到每个任务的 test.txt，例如 t1 的 test.txt，只有 all_task_test.txt。所有任务是否共享相同的 all_task_test.txt？这是否意味着我需要将相同的数据放入 t(1,2,3,4)/images/test 中？
> 
> 正确的

那么按照./split划分的t(1,2,3,4) task下的test images数量分别为4952、1914、1642、1738是不是不重要了？ 可以直接将这四个任务下的test数据集替换根据all_task_test划分的为9119个数量的图片
